### PR TITLE
Sanitise keywords

### PR DESCRIPTION
**Pull request by bpathak-ons**
### What

* Sanitise keywords being returned from APIs

### How to review

* check if code make sense and CI passes

### Who can review

* Anyone

### CODE REVIEW

File: handler/handler.go [hunk 1]
```diff
@@ -67,6 +71,25 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, event *models.Cont
...
+		for j := range strArray {
+			validKeywords = append(validKeywords, strArray[j])
+		}
+	}
+
+	if len(validKeywords) < 5 {
+		return validKeywords
+	} else {
+		return validKeywords[:5]
+	}
```
Comment (daiLlew) [inline handler/handler.go:94]: Can you update this so max valid keyring size is var or const please.

Also `else` not needed here you can just do:
```
    if len(validKeywords) < 5 {
        return validKeywords
    }
    return validKeywords[:5]
```

File: handler/handler.go [hunk 2]
```diff
@@ -67,6 +71,25 @@ func (h *ContentPublishedHandler) Handle(ctx context.Context, event *models.Cont
...
+// incoming keywords validation
+func ValidateKeywords(keywords []string) []string {
+
+	var strArray []string
+	validKeywords := make([]string, 0)
+
+	for i := range keywords {
+		strArray = strings.Split(keywords[i], ",")
+		for j := range strArray {
+			validKeywords = append(validKeywords, strArray[j])
```
Comment (daiLlew) [inline handler/handler.go:86]: Not sure what the requirements are but should we `trim` the keywords to remove any leading/trailing whitespace?

File: handler/handler_test.go [hunk 3]
```diff
@@ -218,6 +218,38 @@ func TestHandlerForZebedeeReturningAllFields(t *testing.T) {
...
+		Convey("When passed to validate the keywords", func() {
+			actual := handler.ValidateKeywords(testKeywords)
+
+			Convey("Then the same keywords should be returned", func() {
+				expectedKeywords := []string{"testkeyword1", "testkeyword2", "testkeyword3", "testKeywords4"}
+				So(actual, ShouldResemble, expectedKeywords)
+			})
+		})
+	})
+}
```
Comment (daiLlew) [inline handler/handler_test.go:251]: Can you also add a test to cover keyword empty?

File: handler/handler.go (hunk 2 above)
Comment (bpathak-ons) [reply handler/handler.go:86]: good one, test included. Thanks

**Review by daiLlew — APPROVED** [review]
